### ISSUE TITLE
unable to run apache2ctl configtest

### ISSUE THREAD

**Issue by gabx** [letsencrypt/letsencrypt#644]
On Fedora 22 running nginx server:

```
(my_env)# ./bin/letsencrypt -d example.com -d www.example.com
```
Then go to the installer window. Enter email, agree,  then it finished with this error:
```
 Unable to run the command: apache2ctl configtest  
```
**apache2ctl** does not exist, but **apachectl** indeed.

**Comment by jdkasten** [comment]
The code has not been packaged or prepared for Fedora yet.

**Comment by kuba** [comment]
Could you please try out `letsencrypt --apache-ctl apachectl ...` in the meantime?

**Comment by gabx** [comment]
I just ``` # ln -s /usr/sbin/apachectl /etc/letsencrypt/apache2ctl```

As a side note, certs are signed **happy hacker fake CA**. I supposed the reason is Let's Encrypt is not yet open?

**Comment by jdkasten** [comment]
Yes, that is correct.  https://letsencrypt.org/faq/

**Comment by jdkasten** [comment]
Glad you found a solution @gabx.  We will need to make sure that plugin initialization failures (apache2ctl) don't cause problems for people who don't intend to use the plugin.

**Comment by GabLeRoux** [comment]
I am experiencing this even when using `2  Nginx Web Server (nginx)`. Plus

```
Unable to run the command: apache2ctl configtest
Nginx Restart Failed!
nginx: [emerg] "error_log" directive is duplicate in
/etc/nginx/sites-enabled/<snip>.com.conf:13
```

Here's my `/etc/nginx/sites-enabled/<snip>.com.conf`:
```
server {
    listen *:80;
    server_name <snip>.com www.<snip>.com;
    index index.html;
    access_log /var/log/nginx/<snip>.com.access.log;
    error_log /var/log/nginx/<snip>.com.error.log;

    location / {
        root /var/www/<snip>.com/;
    }
}
```

I tried again with `sudo ./venv/bin/letsencrypt -vvvvvvv auth` and it seemed to go further, but will open a different issue when I have some more time.

**Comment by pde** [comment]
We need to fix this for 1.0.  We should try a few different commands, and error out only if none of the common recipes work.

**Comment by jdkasten** [comment]
This is old. Apache has been patched to return a No installation error.

Only Debian style configurations are supported at this time. Additional dev
resources will be needed to port it to other OSes.
